corehq/apps/export/utils.py

```diff
@@ -88,6 +88,20 @@ def _convert_index_to_path(index):
         return ['data'] + index.split('.')[1:]
 
 
+def _convert_index_to_path_nodes(index):
+    from corehq.apps.export.models.new import MAIN_TABLE
+    from corehq.apps.export.models.new import PathNode
+    if index == '#':
+        return MAIN_TABLE
+    elif _is_repeat(index):
+        return [PathNode(name='data')] + [
+            PathNode(name=n, is_repeat=True)
+            for n in _strip_repeat_index(index).split('.')[1:]
+        ]
```

Comment:
sounds good. yeah the other tables look similar to `index`. i'd imagine it's just a few more if cases